cmd/find-main.go

```diff
@@ -193,11 +212,52 @@ func mainFind(ctx *cli.Context) error {
 	args := ctx.Args()
 
 	if !ctx.Args().Present() {
-		args = []string{"."}
+		args = []string{"./"}
 	}
 
 	clnt, err := newClient(args[0])
 	fatalIf(err.Trace(args...), "Unable to initialize `"+args[0]+"`")
 
-	return doFind(args[0], clnt, ctx)
+	var (
+		olderThan time.Time
+		newerThan time.Time
+	)
+	if ctx.String("older") != "" {
+		olderThan, err = parseTime(ctx.String("older"))
+		fatalIf(err.Trace(ctx.String("older")), "Unable to parse input threshold value into time.Time")
+	}
+	if ctx.String("newer") != "" {
+		newerThan, err = parseTime(ctx.String("newer"))
+		fatalIf(err.Trace(ctx.String("newer")), "Unable to parse input threshold value into time.Time")
+	}
+
+	var e error
```

Comment:
Why can't we use already declared 'err' variable?

Reply:
they both have different type.